tests/unit/src/cheerioManipulation.test.js

```diff
@@ -271,6 +273,17 @@ describe('Cheerio Manipulation', () => {
         });
     });
 
+    test('Adds values into TLV DOM', async () => {
+      const wrapper = render(SeveralInputs);
+      await userEvent.click(wrapper.container.querySelector('[data-test="button"]'));
```

Comment:
Just to break it up. One idea per line.
```suggestion
      const button = wrapper.container.querySelector('[data-test="button"]');
      await userEvent.click(button);
```